--- /home/nomumu/Arduino/libraries/ros2arduino/src/ros2/topic_id_number.h	2020-05-05 09:08:53.675889538 +0900
+++ topic_id_number.h	2020-05-05 08:52:35.255894401 +0900
@@ -64,7 +64,12 @@ enum TopicIdNumbering
   /* turtlebot3_msgs */
   TURTLEBOT3_MSGS_SOUND_ID,
   TURTLEBOT3_MSGS_VERSION_INFO_ID,
-  TURTLEBOT3_MSGS_SENSOR_STATE_ID
+  TURTLEBOT3_MSGS_SENSOR_STATE_ID,
+
+  /* m5stack_msgs*/
+  M5STACK_MSGS_PLOT_ID,
+  M5STACK_MSGS_STAMP_REQUEST_ID,
+  M5STACK_MSGS_STAMP_DATA_ID
 };
 
 
